edb/repl/__init__.py

```diff
@@ -617,6 +622,7 @@ def _command_list_scalar_types(
             )
         except edgedb.EdgeDBError as exc:
             render.render_exception(self.context, exc)
+            self._command_semicolon_hint(arg)
```

Comment:
Semicolons shouldn't be the cause of an error. They can only be if the input isn't escaped properly. What's the use case that triggers a hint in this except clause?

Reply:
Oh, you're right in case of `\l` commands. The only way it can cause an error is in `\d` because the `InvalidReferenceError` is raised for names that can't be found.

I've removed this from lists' error handling.